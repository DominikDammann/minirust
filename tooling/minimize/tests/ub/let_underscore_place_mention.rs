fn main() {
    let p = {
        let b = 42;
        &b as *const i32 as *const (u8, u8, u8, u8)
    };
    unsafe {
        // Due to the projection, this is not fine.
        // (FIXME: the error is pretty bad, it doesn't mention "offset"/projection...)
        let _ = (*p).1;
    }
}
