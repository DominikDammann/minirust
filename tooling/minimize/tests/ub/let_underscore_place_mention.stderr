fatal error: UB: dereferencing pointer to dead allocation
